pyproject.toml

```diff
@@ -0,0 +1,83 @@
+[project]
+name = "HOTOSM Tasking Manager"
+version = "0.1.0"
+description = "The tool to team up for mapping in OpenStreetMap"
+authors = [
+    {name = "HOT Sysadmin", email = "sysadmin@hotosm.org"},
+]
+dependencies = [
```

Comment:
Dev dependencies should be separated here. Once #5642 is merged, we can use it as a reference since it has a separation of dependencies in requirements.txt.   I think only direct dependencies will be listed in pyproject.toml because pdm.lock will handle all indirect dependencies.

Reply:
We can do it in another PR if  we are merging this ahead of https://github.com/hotosm/tasking-manager/pull/5642.